ceph_deploy/tests/test_cli_new.py

```diff
@@ -19,6 +21,18 @@ def test_help(tmpdir, cli):
     assert 'optional arguments' in result
 
 
+def test_bad_no_mon(tmpdir, cli):
+    with pytest.raises(cli.Failed) as err:
+        with cli(
+            args=['ceph-deploy', 'new'],
+            stderr=subprocess.PIPE,
+            ) as p:
+            result = p.stderr.read()
+    assert 'usage: ceph-deploy new' in result
```

Comment:
multiple assertions in unit tests are not ideal because when it fails, `bad_no_mon` will not map to the exit status or 'too few arguments' or 'usage:'.

If there was a unit test for each (or for all the cases you deem useful) then it would be obvious and succinct:

    TestMonSubcommand::test_not_enough_arguments
    TestMonSubcommand::test_non_zero_exit_status
    TestMonSubcommand::test_show_usage_example

Would make it easy to tell what is going on and to keep the granularity